src/views/accounts/AccountChangeSecret/Executions/AccountChangeSecretExecutionList.vue

```diff
@@ -21,7 +22,21 @@ export default {
       url: `/api/v1/accounts/change-secret-executions/?${params}`,
       detailRoute: 'AccountChangeSecretExecutionDetail',
       automationRoute: 'AccountChangeSecretDetail',
-      resource: 'changesecretexecution'
+      resource: 'changesecretexecution',
+      customActions: {
+        name: 'record',
+        title: this.$t('Record'),
+        can: this.$hasPerm('accounts.view_changesecretrecord'),
+        callback: function({ row }) {
+          return this.$router.push({
+            name: 'AccountChangeSecretList',
+            query: {
+              tab: 'ChangeSecretRecord',
+              execution_id: row.id
+            }
+          })
+        }
+      }
     }
   }
 }
```

Comment:
In most cases, there seem to be no errors or irregularities with the given code snippet. However, it's important not to rely solely on these snippets without running tests that replicate this behavior fully within our real applications because some changes might need specific context.

The only major issue I can notice is missing a trailing comma:

```jsx
export default {
```

Please confirm whether you have run all necessary checks for bugs and ensure that your app's state transitions are correctly implemented according to expected patterns.